src/gluonts/model/forecast.py

```diff
@@ -613,7 +613,7 @@ class OutputType(str, Enum):
 
 
 class Config(pydantic.BaseModel):
-    num_eval_samples: int = pydantic.Schema(100, alias="num_samples")
+    num_samples: int = pydantic.Schema(100, alias="num_samples")
```

Comment:
To ensure backwards compatibility, this should be:

```suggestion
    num_samples: int = pydantic.Schema(100, alias="num_eval_samples")
```